plugins/node/opentelemetry-instrumentation-cassandra/test/cassandra-driver.test.ts

```diff
@@ -53,9 +56,10 @@ function assertSpan(
   span: ReadableSpan,
   name: string,
   query?: string,
-  status?: SpanStatus
+  status?: SpanStatus,
+  customAttributes?: Attributes
 ) {
-  const attributes = {
+  const attributes: any = {
```

Comment:
nit: Instead of `any` you can use `Attributes`. Not sure about it, but I think that the `Attributes` interface was introduce in `@opentelemetry/api@1.1.0`, and since this instrumentation supports 
```json
"peerDependencies": {
  "@opentelemetry/api": "^1.0.0"
},
```

I think we might need to use `SpanAttributes`